main.ledger

```diff
@@ -6842,6 +6862,31 @@
     Liabilities:Reimbursement:Zach Latta
     ; Receipt: 1c8b55868492dfdba4421754285afcc6.pdf
 
+2017/12/03 Gusto
+    Expenses:Operating:Software                         $63.00
+    Assets:Chase:Checking
+    ; Receipt:
+
+2017/12/05 Deposit
```

Comment:
Should be in the same format as other donations. This is from a new donor, so you'll have to increment for them.